### PR TITLE
Universal/Post-processing tweaks

### PR DESCRIPTION
**Pull request by Chman**
### Purpose of this PR

- Better GLES2 warnings in the Editor
- Fixed post-processing being enabled in preview windows
- Fixed post-processing being enabled if a camera doesn't have additional data attached

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [x] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

---
### Overall Product Risks
**Technical Risk**: None

**Halo Effect**: None

### CODE REVIEW

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +57,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed fp16 overflow in Switch in specular calculation
 - Fixed shader compilation errors for Android XR projects.
 - Updated the pipeline Asset UI to cap the render scale at 2x so that it matches the render pipeline implementation limit.
+- Fixed shader compilation errors for Android XR projects.
+- Updated the pipeline Asset UI to cap the render scale at 2x so that it matches the render pipeline implementation limit.
```
Comment (Katstodian) [inline com.unity.render-pipelines.universal/CHANGELOG.md]: Is this via a backport? We're not supposed to change the changelog for previously released packages.

Comment (Chman) [reply com.unity.render-pipelines.universal/CHANGELOG.md]: It's a merge conflict fix, these two lines were moved after I pushed my PR. See the [current file on master](https://github.com/Unity-Technologies/ScriptableRenderPipeline/blob/master/com.unity.render-pipelines.universal/CHANGELOG.md).

Comment (Chman) [reply com.unity.render-pipelines.universal/CHANGELOG.md]: cc @sebastienlagarde [moved them](https://github.com/Unity-Technologies/ScriptableRenderPipeline/commit/fe8491c00be55b5fe03120a387f61f4f6ba59a22#diff-279e9e1a7b1750dc7ce092e1b64a0016) during the version bump to 7.1.1 for some reason.

**Comment by Chman** [comment]
Ref images for 014 have been updated as they were wrong (this PR fixes a bug that affected this test).

**Pull request merged**